src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $item-gap;
+  width: ($item-size * 8) + ($item-gap * 6);
+  margin: 0 auto;
+  padding-top: 50px;
+
+  @each $day, $index in $days {
+    &--start-day--#{$day} &__day:first-child {
+      margin-left: calc((#{$item-gap} + #{$item-size}) * #{$index});
+    }
+  }
+
+  @for $monthDay from 28 through 31 {
+    $monthDayNew: $monthDay + 1;
+
+    &--month-length--#{$monthDay} :nth-child(n + #{$monthDayNew}) {
+      display: none;
+    }
+  }
+
+  &__item {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    @include itemSize($item-size);
+    font-size: 30px;
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: background-color 1s, transform 0.5s;
```

Comment:
Both of the properties should be animated with the duration of half a second.
![image](https://user-images.githubusercontent.com/111275759/194116007-923aca51-7ece-417d-8dab-ec4ea5233c32.png)

```suggestion
    transition: background-color 0.5s, transform 0.5s;
```

Reply:
And separate your css code with empty lines, so it wil be easier to read it